app/models/event/course_record.rb

```diff
@@ -192,6 +196,14 @@ def set_defaults
     true # ensure callback chain continues
   end
 
+  def duration_in_days?
+    return !duration_in_hours?
+  end
+
+  def duration_in_hours?
+    return sk? || tp?
```

Comment:
Gleiche Begründung wie bei `duration_in_days?`

```suggestion
    sk? || tp?
```